stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/notifications/NotificationHandler.kt

```diff
@@ -114,6 +42,11 @@ public interface NotificationHandler : NotificationPermissionHandler, MediaNotif
      * @param localUser The local Stream user.
      * @param onUpdate Callback to be called when the notification is updated.
      */
+    @Deprecated(
+        level = DeprecationLevel.ERROR,
+        message = "This method is deprecated. Use the getNotificationUpdates method in the NotificationHandler interface instead.",
```

Comment:
I think the message needs a bit of correction, the alternate `method` on the message text an on repaceWith text points to different method